tgui/packages/tgui-panel/audio/NowPlayingWidget.js

```diff
@@ -6,33 +6,51 @@
 
 import { toFixed } from 'common/math';
 import { useDispatch, useSelector } from 'common/redux';
-import { Button, Flex, Knob } from 'tgui/components';
+import { Button, Collapsible, Flex, Knob } from 'tgui/components';
 import { useSettings } from '../settings';
 import { selectAudio } from './selectors';
 
 export const NowPlayingWidget = (props, context) => {
-  const audio = useSelector(context, selectAudio);
-  const dispatch = useDispatch(context);
-  const settings = useSettings(context);
-  const title = audio.meta?.title;
+  const audio = useSelector(context, selectAudio),
```

Comment:
Do not use commas to bundle together consts, keep them each with their own const keywords.

Reply:
Might i ask why? Im not arguing, but im asking how come? I have been programming JS stuff for a while now and have always done it this way and it makes me curious why you say otherwise?

Reply:
From what i know 
`const var1;
const var2;
const var3;`
is the Exact same as 
`const var1,var2,var3`
it just cleans things up and has things displayed more neatly as having all the extra characters for the other consts is unnecessary where logically both function the same